src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -75,12 +78,18 @@ class Microsoft_Launcher : public PowertoyModuleIface
     // Handle to event used to invoke the Runner
     HANDLE m_hEvent;
 
+    std::shared_ptr<Logger> logger;
+
 public:
     // Constructor
     Microsoft_Launcher()
     {
         app_name = GET_RESOURCE_STRING(IDS_LAUNCHER_NAME);
         app_key = LauncherConstants::ModuleKey;
+        std::filesystem::path logFilePath(PTSettingsHelper::get_module_save_folder_location(this->app_key));
+        logFilePath.append("logging.txt");
+        logger = std::make_shared<Logger>("launcher", logFilePath.wstring(), PTSettingsHelper::get_log_settings_file_location());
+        logger -> info("Launcher object is constructing");
```

Comment:
regarding the formatting - this stuff could be done automatically by executing `powershell src\codeAnalysis\format_sources.ps1` command from a VS cmd shell before commiting your changes. I guess we could also add precommit hook for it (@enricogior new task? 😃 )

Reply:
btw, we could also turn on automatic source formatting based on .clang-format in VS settings!

Reply:
I ran that command and it seems like everything is already formatted.
 
![image](https://user-images.githubusercontent.com/17161067/99293247-0ebf2c00-284b-11eb-9fed-8b080a94dbee.png)

Reply:
it only detects the modified files in your work dir using git, so you'd need to amend the last commit or run the script with -all (it's slow though)

Reply:
I have just run it with 'all' flag and do not see any files modified.

Reply:
Script assumes that it is run from src folder.

Reply:
done